src/main/java/com/kylenicholls/stash/parameterizedbuilds/ciserver/CIServerFactory.java

```diff
@@ -41,7 +41,14 @@ public static CIServer getServer(String pathInfo, Jenkins jenkins,
     private static Server getServerFromMap(Map<String, String[]> parameters) {
         boolean jenkinsAltUrl = parameters.get("jenkinsAltUrl") != null
                 && "on".equals(parameters.get("jenkinsAltUrl")[0]) ? true : false;
+        boolean jenkinsCSRF = false;
+        try {
+            jenkinsCSRF = parameters.get("jenkinsCSRF") != null
+                    && "on".equals(parameters.get("jenkinsCSRF")[0]) ? true : false;
+        } catch (Exception e) {
```

Comment:
Empty `catch (Exception e)` blocks scare me. What sort of exception do you expect to occur? If possible, I think we should make the error caught be more explicit and add some logging about what happened.

Reply:
Noy sure it's necessary it's mainly for backward compatibility because the map in the database does not contain ani csrf boolean at first.